app/views/result_filters/location/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= content_for :page_title, "Find courses by location or by training provider" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(results_path(request.query_parameters), "Back to search results") %>
+<% end %>
+
+<% if flash[:error] %>
```

Comment:
I feel like this should go into a partial now. We are using it in number of places. Not a blocker for this PR.

Reply:
I've added it as a partial in the qualification filter now. I'll update the others when it gets approved.